tiled/_tests/test_sync.py

```diff
@@ -88,6 +90,35 @@ def test_copy_internal():
             read(dest, strict=True)
 
 
+def test_copy_skip_conflict():
+    with client_factory() as dest:
+        with client_factory() as source:
+            populate_internal(source)
+            copy(source, dest)
+            copy(source, dest, on_conflict="skip")
+            assert list(source) == list(dest)
+            assert list(source["c"]) == list(dest["c"])
+            read(dest, strict=True)
```

Comment:
These assertions are really not germane to this test, a better test case would be to try to copy *different* data in the second copy and assert that the data in `dest` matches the first and not the second (really a test for a future `overwrite` option for `on_conflict`).  This could wait until we have `overwrite`.

Reply:
My two cents: fine to wait.